tests/test_ppod.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from lambdas.ppod import lambda_handler
+
+
+def test_ppod_configures_sentry_if_dsn_present(caplog, monkeypatch):
+    monkeypatch.setenv("WORKSPACE", "test")
+    monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
+    caplog.set_level(logging.INFO)
+    output = lambda_handler({}, {})
+    assert (
+        "Sentry DSN found, exceptions will be sent to Sentry with env=test"
+        in caplog.text
+    )
+    assert output == "lambda"
```

Comment:
Testing the lambda result is good but I would do it in a separate test function that's explicitly for that (that test will then get updated in the next PR, without impacting these two).